src/orquestra/sdk/_base/cli/_dorq/_arg_resolvers.py

```diff
@@ -115,18 +118,43 @@ def __init__(
         self._wf_run_repo = wf_run_repo
         self._prompter = prompter
 
+    def _prepare_wf_list_for_prompt(self, wfs):
+        # sort wfs by submission date. Take into account when there is no start_time
+        wfs.sort(
+            key=lambda wf: wf.status.start_time
+            if wf.status.start_time
+            else datetime.datetime.fromtimestamp(0),
+            reverse=True,
+        )
+        # Create labels of wf that are printed by prompter
+        # Label is <wf_id> <start_time> tabulated nicely to create good-looking table
+        labels = [
```

Comment:
**suggestion**: to me generating labels feels closer to the presentation layer than the data layer. Wanna move it to `_ui._presenters`? Bonus: you'll be able to reuse already existing formatting helpers for datetimes and `_format_tasks_succeeded`